assets/blocks/digital-wallets/button-component.js

```diff
@@ -171,6 +179,7 @@ const ButtonComponent = () => {
 			)}
 			{!isGooglePayDisabled && (
 				<div
+					style={{ height: `${buttonHeight}px` }}
```

Comment:
Curious why the Google Pay button needs `px` added but the Apple Pay one doesn't?

Reply:
Using a number instead of a string implies the `px` unit. As for which one to use, I guess it's a matter of preference, maybe the latter is preferable since it's very clear that we are using pixels 🤷🏼 .

Reply:
That's actually an oversight on my part, good spot!

Reply:
Should be fixed now, both should use px values